doc/DeveloperGuide.md

```diff
@@ -59,7 +59,9 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | user | add a new person |
 `* * *` | user | delete a person | remove entries that I no longer need
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
+`* * *` | user | edit a person's information | update any details easily when changes occur in one step
 `* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
+`* *` | user | display people with certain tags | locate a group of people easily 
```

Comment:
You can think of more specific class of users; this helps you to narrow down the target audience for your application better.